tests/test_api.py

```diff
@@ -1910,3 +1923,417 @@ def test(self):
             RedisSubscriptionDataStore(redis_client, self.dataset, partition,).all()
             == []
         )
+
+
+class TestExtraAPI(BaseApiTest):
+    """ These are tests for features that are not supported in SnQL. """
```

Comment:
I'm assuming this is all copy/paste so skipping over

Reply:
Actually why does it need move at all? There's a lot of duplication between the two classes now. Isn't it just the post method that differs very slightly for the methods that we test both legacy and snql vs only legacy?

Reply:
These are all tests that were doing something that SnQL straight won't allow. E.g. having a query with no selected columns. It's not possible to run these with SnQL so I moved them here so we wouldn't lose test coverage.

Reply:
I'm going to rename this LegacyTests or something like that to make it more clear. I must have been tired when I came up with that class name.

Reply:
I didn't mean to delete it entirely or anything like that. I just wondered if we can't just pass a parameter to the fixture or something to tell it to include/exclude the Sqnl related variants as required. Otherwise we end up with a lot of duplication, and then we might end up with the same issues in `test_transactions_api`, `test_sessions_api`, `test_discover_api`, `test_outcomes_api`, etc.

Reply:
I'll clean up the duplication as well.

Reply:
> I just wondered if we can't just pass a parameter to the fixture or something to tell it to include/exclude the Sqnl related variants as required.

Isn't that what I did by moving the tests to a new class? If you want to run tests without the SnQL tests, you don't include the fixtures in the first place.